packages/cms-cli/lib/prompts.js

```diff
@@ -54,7 +51,7 @@ const PORTAL_API_KEY = {
   name: 'apiKey',
   message: 'Enter the API key for your portal:',
   validate(val) {
-    if (!API_KEY_REGEX.test(val)) {
+    if (!val.match(API_KEY_REGEX)) {
```

Comment:
This will also work but what you had before (`rgx.test(str)`) is preferable since it always returns a bool.